README.md

```diff
@@ -223,6 +224,22 @@ Note, this will only specifically enable the `RequestLogger`.
 - [Updating diagrams](/docs/updating-diagrams.md)
 - [Setting up a CircleCI context for deployment](/docs/setting-up-circleci-context-for-deployment.md)
 
+## External dependencies
+### This solution is dependent on
+- NOMIS database via the "Prison API"
+- Prisoner Offender Search
+- Probation Offender Search
+- HMPPS Auth
+- Cloud Platform
+- Azure Application Insights
+- Sentry
```

Comment:
> Cloud Platform
> Azure Application Insights
> Sentry

I wonder if we depend on these or just use these? Could it be worth adding another heading which might be something like "This solution uses..."